### ISSUE TITLE
Suggestion: an option to disable the home feed

### ISSUE THREAD

**Issue by drjedd** [indywidualny/FaceSlim#87]
Hello there,

First of all, thanks for making this app! It works neatly!

I'm a facebook user by constraint: I use the messaging system to stay in touch with long distance friends and organize group-ups, that's also my source for events to go to, etc.

But there's one thing I really dislike about facebook: the fact that before you can even do all this interesting stuff, you are presented with an endless feed of unimportant posts about people's lives and funny cat videos, etc.

So about 8 months ago I installed uBlock on my desktop browser with some custom rules to prevent facebook from displaying the news feed. I can still access all the important stuff and specific posts if I'm involved in them (tagged or mentioned in a comment), but I never get side-tracked and I'm using facebook as an efficient tool for my work and (actual) social life.

I wish I could do the same on my mobile too. And I wish other people had the option to try this healthier use of facebook. That's why I would love to see an option to disable the news feed altogether, or maybe make it the last tab and not the default go-to, so you can take care of the important stuff before your eyes get baited by a catchy title of picture.

I know it's not an easy task but it could really change a lot of people's lives so please consider it :)

If it's any help, here are the ublock custom filters I've got right now:

```
facebook.com##div#stream_pagelet
facebook.com##div#pagelet_trending_tags_and_topics
facebook.com##div#pagelet_ego_pane
```

So basically just hiding any `<div>` with those classes. It's sort of a quick fix but it does the job!

Thank you very much! Have a nice day and thanks again for making this app!

jedd

**Comment by indywidualny** [comment]
Ok. It's a strange request but I can try. Could you find those divs here too? https://m.facebook.com/

I will add a similar rule.

**Comment by drjedd** [comment]
Thanks for considering it! :)

It seems like the mobile version use a different template with different class names. Blocking/hiding the <div> with the id "m_newsfeed_stream" would achieve the same effect.

I don't think that request is that strange :) You should try this yourself for a few days and enjoy the freedom! :P 

Anyways, thanks again!

**Comment by indywidualny** [comment]
![2015-12-22-005737_1920x1080_scrot](https://cloud.githubusercontent.com/assets/9354672/11944536/4380db70-a847-11e5-873d-ab671e3eb07e.png)

Not bad :) It gonna be introduced in the next release.

**Comment by indywidualny** [comment]
One more thing. I have no idea what description I should add to this feature. Halp :D

**Hide News Feed**
Some description, etc, etc.

**Comment by juzman** [comment]
Nice addition! Does this also hide the "people you may know" section, I
hate that bit.

For description, something like:

This option cleans the news feed by hiding everything that you're not
tagged in.

**Comment by indywidualny** [comment]
I will also hide "people you may know" section although I have to locate it and it's not visible so often.
I'm not sure that posts you are tagged in are visible too. It need to be tested. For now I can only notice... nothing :)

**Comment by drjedd** [comment]
@juzman it will hide the posts you're tagged in from the news feed for sure (it hides the actual container for the news feed), but you should still get the notification and clicking that notification should display the mentioned post/picture/etc. without problem because it's not using that div id.

@indywidualny looking forward to next version then! I think you could add "[...] to avoid sidetracking/procrastination" to the description!

Thanks again!

**Comment by juzman** [comment]
So essentially the news feed is always blank?

**Comment by indywidualny** [comment]
Yes. Ok, it's gonna be released soon. I cannot find div for "people you may know". It will be done later.

**Comment by drjedd** [comment]
Thank you. Really looking forward to this one. Every time I launch Facebook I see someone bragging about useless shit and I go straight to F droid to check for updates :P really hope it is soon.

**Comment by indywidualny** [comment]
Please give me a few days. I'm going to solve #24 and a minor bug tomorrow and I'm ready to release :)

**Comment by juzman** [comment]
Sounds good, cannot wait for new update. :-)

**Comment by drjedd** [comment]
Fantastic, enabled it straight away. I can enjoy a much better experience now! Thank you how so much!!!

**Comment by landry314** [comment]
I also love this feature as the newsfeed is addictive and don't want it in my face when I try to search for events or send messages.

One problem though: there is a small delay when opening the app when you can still see the newsfeed before it is hidden. I don't want to ever see the newsfeed. A change in the order of rendering might fix this, I don't know.

Maybe it is just that I have an older phone... does anyone else have this problem?
